ui-tests/src/test/java/io/syndesis/qe/steps/other/GMailSteps.java

```diff
@@ -0,0 +1,44 @@
+package io.syndesis.qe.steps.other;
+
+import com.google.api.services.gmail.model.Message;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import io.syndesis.qe.utils.GMailUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import java.io.IOException;
+import java.util.List;
+
+@Slf4j
+public class GMailSteps {
+
+    @Then("^.*send an e-mail$")
+    public void sendMail() {
+        GMailUtils.sendEmail("me", "jbossqa.fuse@gmail.com", "syndesis-tests", "Red Hat");
+    }
+
+    @And("^delete emails from \"([^\"]*)\" with subject \"([^\"]*)\"$")
+    public void deleteMails(String from, String subject) {
+        GMailUtils.deleteMessages(from, subject);
+    }
+
+    @And("^check that email from \"([^\"]*)\" with subject \"([^\"]*)\" and text \"([^\"]*)\" exists$")
+    public void checkMails(String from, String subject, String text) {
+        try {
+            List<Message> messages = GMailUtils.getMessagesMatchingQuery("me", "subject:" + subject + " AND from:" + from);
+            Assertions.assertThat(messages.size()).isEqualTo(1);
+
+            MimeMessage mime = GMailUtils.getMimeMessage(messages.get(0).getId());
+            Assertions.assertThat(mime.getSubject()).isEqualToIgnoringCase(subject);
+            //note that getContent() works here on because of specific message we sent (nothing special inside of the message,
+            //no attachment etc. otherwise extracting text should be handled differently
+            Assertions.assertThat(mime.getContent().toString()).isEqualToIgnoringCase(text);
```

Comment:
optional: I like to static import assertThat so I don't need to write Assertions. everytime :-)